libs/core/src/components/pds-avatar/pds-avatar.scss

```diff
@@ -20,15 +20,8 @@ div {
     border-radius: 12px;
   }
 
-  // Remove when FF has support for :has
-  :host(.pds-avatar--has-image) & {
-    background: transparent
-  }
-
-  @supports selector(:has) {
-    &:has(img) {
-      background-color: transparent
-    }
+  &:has(img) {
```

Comment:
This comment was left on your Slack thread, but I'll put it here so it doesn't get lost.

While I love the enthusiasm to use the selector, I have one concern. What is the fallback or outcome should a customer not have a supported browser installed?